### PR TITLE
[author] Update simple-icons package.json

### LINKED ISSUE #12885: [Request] Update simple-icons [author]
`simple-icons` changed hands (and git repo) a while ago. It further seems like the CDNJS version claims to be at `3.0.1`, a version that [never existed](https://github.com/simple-icons/simple-icons/commits/develop/package.json) - the currently newest version is `1.6.7`. This is likely why the git autoupdating fails.

**Git repo changed:**  
`https://github.com/danleech/simple-icons.git` -> `https://github.com/simple-icons/simple-icons.git`

**Version should be fixed:**
`3.0.1` -> `1.6.7`

**Source issue:**
simple-icons/simple-icons#891

The homepage remains the same.

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/61156907-request-update-simple-icons-author?utm_campaign=plugin&utm_content=tracker%2F32893&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F32893&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by birjolaxew**
Switches from git to NPM auto-update
Updates repo to reflect new ownership

I was unsure if the version should be updated, given that the contribution guidelines explicitly tell me not to - but if your autoupdater checks if the new version > old version, then it should be manually changed (as the proper version is `1.6.7` and your current version claims to be `3.0.1`)

I was also unsure if the existing version folders should be deleted. Please do comment if they should.

Closes #12885

Pull request for issue: #12885 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

# Profile of the lib
 * Git repository (required): https://github.com/simple-icons/simple-icons
 * Official website (optional, not the repository): https://simpleicons.org
 * NPM package url (optional): https://www.npmjs.com/package/simple-icons
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 196
   - Count of stars: 6311
   - Count of forks: 709
 * NPM download stats (optional):
   - Downloads in the last week: 872

# Essential checklist
 * [x] I'm the author of this library (one of multiple maintainers)
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [x] Doesn't already exist / has duplicate issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
 * [x] The parent of the commit(s) in the PR is not older than 3 days.
 * [x] Pull request is sent from a non-master branch with a meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/simple-icons/package.json [hunk 1]
```diff
@@ -1,34 +1,26 @@
...
     "icons",
-    "SVG",
-    "PNG",
-    "image"
+    "svg"
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/danleech/simple-icons.git"
+    "url": "git+ssh://git@github.com/simple-icons/simple-icons.git"
```
Comment (extend1994) [inline ajax/libs/simple-icons/package.json]: Let's use `https://github.com/simple-icons/simple-icons.git` or cdnjs website will fail to bring users to the expected web page.

**Comment by birjolaxew** [comment]
@extend1994 Sure, if you have a way to pull the repo in reasonable time that'd be appreciated. Working with the web UI isn't exactly great.

I'd also like to explicitly confirm that leaving the version at `3.0.1`, and leaving the version folders as-is, is the right way to go when the package should be downgraded to `1.6.7`. Is that correct?

**Comment by extend1994** [comment]
@birjolaxew No problem. 
And **correct**, `version` change follows the latest version of the lib with its assets. 
As usual, we don't remove the hosted version on cdnjs to prevent anyone from being unable to access it. But the situation here is that you said v3 actually never existed, I am afraid I can't make that decision.
All in all, I will help do the changes I mentioned in my review and let @PeterDaveHello, who added simple-icons v2 & v3, to handle others parts.

**Comment by extend1994** [comment]
@PeterDaveHello I help separate the commits, please help review this PR, thanks!

**Comment by PeterDaveHello** [comment]
v3.0.1 was added by the bot: 96f0a7ae3c1d08a241e5cfe99db2722d05efcf55, so I believe that's from the git repo.

**Comment by PeterDaveHello** [comment]
@birjolaxew would you like to confirm?
```
simple-icons $ git log -1 v3.0.1
commit 8992949d476b2934a42b9dd5c3dbc4e182113e91
Merge: 03d0df7 9bb15ee
Author: Dan Leech <danleech@users.noreply.github.com>
Date:   Sun Apr 30 16:50:34 2017 +0100

    Merge pull request #417 from danleech/develop
    
    Adds AddThis and Mastodon
```
```
simple-icons $ git log -1 3.0.1
commit 0a499fd2945327764e1d68b57597affd55ebef67
Merge: 225c0cc 9149710
Author: Dan Leech <danleech@users.noreply.github.com>
Date:   Thu Apr 27 23:13:09 2017 +0100

    Merge pull request #410 from danleech/develop
    
    Makes icons links download by default
```

**Comment by birjolaxew** [comment]
Those commits [do indeed exist](https://github.com/simple-icons/simple-icons/commit/8992949d476b2934a42b9dd5c3dbc4e182113e91), but they do not have a release version connected to them. In fact, no release tags are present in our git repo:

```
~/dev/simple-icons » git log v3.0.1
fatal: ambiguous argument 'v3.0.1': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'

~/dev/simple-icons » git log 3.0.1
fatal: ambiguous argument '3.0.1': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'

~/dev/simple-icons » git tag && echo "Tags are above (if any)" 
Tags are above (if any)
```

Official releases (and therefore non-1.0.0 release versions) only started happening [after Dan Leech handed us the project and we started publishing to NPM](https://github.com/simple-icons/simple-icons/commit/70bbd06e333c493e9747d9a6ee937deef726289c#diff-b9cfc7f2cdf78a7f4b91a753d10865a2).

**Comment by PeterDaveHello** [comment]
@birjolaxew Maybe they were created and deleted, the commits records come from our auto-update server, which were fetched before the repository transferring.

**Comment by birjolaxew** [comment]
@PeterDaveHello That is very possible.

**Comment by PeterDaveHello** [comment]
@birjolaxew then, I'd say it's not a good idea to "reversioning" as it can lead misunderstanding and confusing ... though it's been awhile ...

**Comment by birjolaxew** [comment]
@PeterDaveHello It doesn't seem to me like Simple Icons has ever had any versions but `1.0.0` prior to us taking over, neither in the form of git tags nor `package.json` (although it is possible that a git tag was pushed and then deleted before we took over, that seems unlike Dan Leech to do - and there is no real way for us to check if it did happen).

The official version has progressed from [`1.0.0`](https://github.com/simple-icons/simple-icons/commit/70bbd06e333c493e9747d9a6ee937deef726289c#diff-b9cfc7f2cdf78a7f4b91a753d10865a2) to the current `1.7.0`. The mismatch is in your repo; I'll let you guys decide how to handle it.

**Comment by PeterDaveHello** [comment]
@birjolaxew we better not to delete any already hosted versions in case to cause breaking changes.
Not sure if it's possible to name the new ones to new-simple-icons, and add the info in readme to guide users to the correct one? What do you think?

**Comment by birjolaxew** [comment]
@PeterDaveHello Not a great solution for obvious reasons, but it isn't a great situation to begin with. We'd be happy to add that info to our readme if it is the solution you feel is best.

**Comment by birjolaxew** [comment]
Pinging other Simple Icons maintainers for their input @ericcornelissen & @davidklebanoff

**Comment by ericcornelissen** [comment]
Don't love the "new-simple-icons" solution, but as @birjolaxew said, if the maintainers of cdnjs.com feel it's the best solution I'm fine with that.

I'm not all too familiar with cdnjs.com, but wouldn't it be possible to deprecate `v3.0.1` (and other versions above `1.7.0`). Not sure if there exists such a feature for cdnjs tho... 🤔

**Comment by PeterDaveHello** [comment]
@ericcornelissen I haven't figured out a good solution yet, but the deprecate could casue breakings of other sites which is not a good idea. Still open for discussion and suggestion, thanks a lot.

**Comment by ericcornelissen** [comment]
@PeterDaveHello I understand it will lead to broken sites, but that's why I would deprecate them now, I don't think we'll be hitting version 3.0.1 anytime soon. So site maintainers, I think, will have plenty of time to update.

However, if you guys do not yet have a deprecated feature I would vote against deprecating the existing versions, since people who decided to use those could not have known the link could be deprecated...

**Comment by birjolaxew** [comment]
Keep in mind that @ericcornelissen probably means deprecation as in "warn users that the version they are using is deprecated, then remove it after some time" - not just straight up removing it, leading to broken sites.

Unfortunately this is difficult to do for a project like cdnjs since you have no record of who uses the library, and you have no way of embedding a `console.log` inside of an SVG file (that is loaded remotely).

I think the best way would be if you could look at your traffic stats and check how many people use any of the incorrect versions (`>= 2`, not `@latest`) to see if there would actually be an impact from removing them. If people are using version-targetted links to embed the icons from cdnjs, then we cannot simply remove those links.

**Comment by ericcornelissen** [comment]
> and you have no way of embedding a console.log inside of an SVG file (that is loaded remotely).

Didn't think about that 😅

**Comment by xtuc** [comment]
The pr doesn't delete any existing files anymore, what prevents from merging?

**Pull request merged**